### PR TITLE
Add celery utility commands

### PR DESCRIPTION
**Pull request by gcapalbo**
@czue

### CODE REVIEW

**Comment by czue** [comment]
nice thanks. looks very useful. build is failing (i think just needs inline import)
```
Traceback (most recent call last):
  File "./manage.py", line 95, in <module>
    execute_from_command_line(sys.argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/vendor/lib/python2.7/site-packages/django/__init__.py", line 20, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/vendor/lib/python2.7/site-packages/django/utils/log.py", line 87, in configure_logging
    logging_config_func(logging_settings)
  File "/usr/local/lib/python2.7/logging/config.py", line 794, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/local/lib/python2.7/logging/config.py", line 559, in configure
    'filter %r: %s' % (name, e))
ValueError: Unable to configure filter 'hqcontext': Cannot resolve 'corehq.util.log.HQRequestFilter': cannot import name Celery
```

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Comment by gcapalbo** [comment]
ok thanks, actually i also have some updates i want to make to this before we merge

**Comment by czue** [comment]
:+1: just ping when you're ready - just glad to have this available

File: corehq/util/celery_utils.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import kombu.five
+import numbers
```
Comment (dimagimon) [inline corehq/util/celery_utils.py]: F401 'numbers' imported but unused

**Comment by gcapalbo** [comment]
ok, this should be good to go now. build should be fixed too, but good to wait for it

File: corehq/util/celery_utils.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+from datetime import datetime
+from time import sleep, time
+
+
+class TaskInfo(object):
+
+    def __init__(self, _id, name, time_start=None):
+        self.id = _id
+        self.name = name
+        # http://stackoverflow.com/questions/20091505/celery-task-with-a-time-start-attribute-in-1970
```
Comment (czue) [inline corehq/util/celery_utils.py]: this is wild

File: corehq/util/celery_utils.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    result = []
+    for task_dict in tasks:
+        result.append(TaskInfo(task_dict.get('id'), task_dict.get('name')))
+    return result
+
+
+def get_scheduled_task_info(tasks):
+    result = []
+    for task_dict in tasks:
+        task_dict = task_dict.get('request', {})
```
Comment (czue) [inline corehq/util/celery_utils.py]: is this just some weird difference in how celery internals represents scheduled tasks?

**Comment by czue** [comment]
this looks amazing, thanks.

File: corehq/util/celery_utils.py (hunk 3 above)
Comment (gcapalbo) [inline corehq/util/celery_utils.py]: yeah, for some reason it's just a different json response

**Pull request merged**